libpod/container_config.go

```diff
@@ -229,6 +229,12 @@ type ContainerNetworkConfig struct {
 	// namespace
 	// These are not used unless CreateNetNS is true
 	PortMappings []ocicni.PortMapping `json:"portMappings,omitempty"`
+	// ExposedPorts are the ports which are exposed but not forwarded
+	// into the container.
+	// The map key is the port and the string slice contains the protocols,
+	// e.g. tcp and udp
+	// These are only set when exposed ports are given but not published.
+	ExposedPorts map[uint16][]string `json:"exposedPorts,omitempty"`
```

Comment:
We should probably wire this into Commit, so the exposed ports are added to committed images. I'm actually kind of amazed this wasn't in before - had to double-check to make sure.

Reply:
Good call, they get commited now.